packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -477,24 +483,44 @@ class _SearchViewRoute extends PopupRoute<_SearchViewRoute> {
     final Size screenSize = MediaQuery.of(context).size;
     final Rect anchorRect = getRect() ?? Rect.zero;
 
-    // Check if the search view goes off the screen.
     final BoxConstraints effectiveConstraints = viewConstraints ?? viewTheme.constraints ?? viewDefaults.constraints!;
-    final double verticalDistanceToEdge = screenSize.height - anchorRect.top;
-    final double endHeight = math.max(effectiveConstraints.minHeight, math.min(screenSize.height * 2 / 3, verticalDistanceToEdge));
     _rectTween.begin = anchorRect;
 
+    double viewWidth = anchorRect.width;
+    viewWidth = clampDouble(viewWidth, effectiveConstraints.minWidth, effectiveConstraints.maxWidth);
```

Comment:
Is this for readability? Couldn't we make this one line? 
```suggestion
    double viewWidth = anchorRect.width;
    viewWidth = clampDouble(viewWidth, effectiveConstraints.minWidth, effectiveConstraints.maxWidth);
```

Reply:
Same for `viewWidth` in other places.

Reply:
Just fixed! Thanks:) !